pkg/providers/cloudstack/testdata/expected_results_main_md.yaml

```diff
@@ -14,6 +14,7 @@ spec:
             provider-id: cloudstack:///'{{ ds.meta_data.instance_id }}'
             read-only-port: "0"
             anonymous-auth: "false"
+            node-labels: cluster.x-k8s.io/failure-domain='{{ds.meta_data.availability_zone}}'
```

Comment:
I think this will trigger a rollingupgrade on all of a management cluster's workload clusters. We should try to avoid that from happening. I'm not sure what the best way of doing that is.

Reply:
We are trying to add label for nodes that is provisioned using eks-a, should this be considered a valid reason for upgrade?